front/src/modules/trainschedule/components/TimetableV2/hooks.ts

```diff
@@ -91,11 +91,11 @@ const useTrainSchedulesDetails = (
         if (scheduledPointsHonoredFilter !== 'both') {
           if (trainSummary.status !== 'success') return false;
 
-          const isHonored = isScheduledPointsNotHonored(trainSchedule, trainSummary);
+          const isNotHonored = isScheduledPointsNotHonored(trainSchedule, trainSummary);
 
           if (
-            (scheduledPointsHonoredFilter === 'honored' && !isHonored) ||
-            (scheduledPointsHonoredFilter === 'notHonored' && isHonored)
+            (scheduledPointsHonoredFilter === 'honored' && !isNotHonored) ||
+            (scheduledPointsHonoredFilter === 'notHonored' && isNotHonored)
```

Comment:
Small bug spotted while testing
```suggestion
            (scheduledPointsHonoredFilter === 'honored' && isNotHonored) ||
            (scheduledPointsHonoredFilter === 'notHonored' && !isNotHonored)
```

Reply:
Done !